pycc/local.py

```diff
@@ -343,6 +344,16 @@ def _build_PNO(self):
         self.eps = eps  # semicananonical local energies
         self.dim = dim  # dimension of local space
 
+        #temporary way to generate make sure the phase factor of Q_ij and L_ij matches with Q_ji and L_ji
+        for i in range(self.no):
+            for j in range(self.no):
+                if i < j:
```

Comment:
Rather than inserting a conditional within the `j` loop, it would be better simply to limit `j` to be less than `i`.